doc/holochain_101/src/zome/entry_type_definitions.md

```diff
@@ -0,0 +1,295 @@
+# App Entry Type Definitions
+
+An "entry" is a data element that an agent authors to their local device, that is then propogated to peers running the same app. The entry is backed by a "chain header", which is the data element used for verification of the integrity of itself, as well as the entry.
+
+Entries are a fundamental, primitive type within Holochain. Entries are an abstraction, they can technically be persisted to a device in a variety of ways, using a variety of databases, which can be as simple as files in the file system.
+
+There are types of entries which cannot be written to the chain by users of an application. These are generally called system entries. They include DNA, and initial Agent entries, which are always the first two entries written to a chain.
+
+There are a special type of entries called App Entries. These are entries which are created through the active use of an application by a user. They must have an entry type which, rather than being system defined, is defined application specific.
```

Comment:
```suggestion
There are a special type of entries called App Entries. These are entries which are created through the active use of an application by a user. They must have an entry type which, rather than being system defined, is defined by the Zome developer.
```